packages/editor/src/components/header/index.js

```diff
@@ -75,6 +78,10 @@ function Header( {
 		};
 	}, [] );
 
+	const canBeZoomedOut = [ 'post', 'page', 'wp_template' ].includes(
```

Comment:
Another case for this issue https://github.com/WordPress/gutenberg/issues/61622

How do we see that longer term?